catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "backstage.io/v1alpha1"
+kind: "Component"
+metadata:
+  name: "ember-redux"
+  description: |
+    Ember bindings for Redux
+
+    During our Ember 4.0 upgrade we ran into some issues with ember-redux, and we had to fork it in order to fix it.
+    These fixes are also available in the upstream repo, but they are not released yet.
+    See this pull request for further details: https://github.com/ember-redux/ember-redux/pull/213
```

Comment:
There's very little space in backstage for displaying the description. See if you (or AI) can make it shorter.

Reply:
Done. Let me know what you think 👍